staging/src/k8s.io/apiextensions-apiserver/hack/build-image.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Copyright 2014 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../../../../..
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+# Register function to be called on EXIT to remove generated binary.
+function cleanup {
+  rm "${KUBE_ROOT}/vendor/k8s.io/apiextensions-apiserver/artifacts/simple-image/apiextensions-apiserver"
+}
+trap cleanup EXIT
+
+pushd "${KUBE_ROOT}/vendor/k8s.io/apiextensions-apiserver"
+cp -v ../../../../_output/local/bin/linux/amd64/apiextensions-apiserver ./artifacts/simple-image/apiextensions-apiserver
+docker build -t apiextensions-apiserver:latest ./artifacts/simple-image
```

Comment:
cc @kubernetes/sig-release on image build … is this intended to be part of the release artifacts?

Reply:
We claim this can run in standalone mode. I think it makes sense to ship a binary just to keep that true.

Reply:
sure, this was more a question about whether this was the best way to build this image... other images have switched to cloudbuild

Reply:
If I understand the concern correctly, apiextensions-apiserver is not part of the release images. Server targets are listed here:
https://github.com/kubernetes/kubernetes/blob/master/hack/lib/golang.sh#L90

If we consider adding the kube-aggregator and apiextensions-apiserver images to the release artifacts, I can help do the job.
This is more of a script to help users build the image themselves, just like what has been done by the kube-aggregator project:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-aggregator/hack/build-image.sh

Reply:
@tpepper could you help take a look at the question?

Reply:
Quibbling here. I agree it would be good to generate (& test?) an image to validate that this can be run in standalone mode. However I do not see any point in actually shipping the image.

Reply:
IIUC, having this build image scripts for standalone uses is good enough, correct?